cli/README.md

```
@@ -160,65 +160,65 @@ Commands follow `carbidecli <resource> [sub-resource] <action> [args] [flags]`.
 
 # Fish
-carbidecli completion fish > ~/.config/fish/completions/carbidecli.fish
+nicocli completion fish > ~/.config/fish/completions/nicocli.fish
 ```
 
 ## Multi-Environment Configs
 
-Each environment (local dev, staging, prod) gets its own config file in `~/.carbide/`:
+Each environment (local dev, staging, prod) gets its own config file in `~/.nico/`:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)